napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -478,21 +492,25 @@ def test_selecting_all_points_with_drag_2d(create_known_points_layer_2d):
     layer.mode = 'select'
 
     # Simulate click
-    event = read_only_event(type='mouse_press', position=known_non_point)
+    event = read_only_event(type='mouse_press', position=(20, 20))
```

Comment:
Why the change from "`known_non_point`"? This is less readable.

Reply:
Using `known_not_point` here was not really correct, cause we're really trying to select by dragging a box that covers all the points. It worked before because `known_not_point` was ouside the points (see comment above to why that was changed).

Basically, this tests does not need to click any `non-point`, it needs to click specifically a point beyond the corner of the bounding box... I could name this in a variable to make it clearer maybe?

Reply:
> I could name this in a variable to make it clearer maybe?

Yeah, that might be nice to do. It's not a blocker to merging, though

Reply:
added variables for clarity!